lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/DefaultRemoteManagerFactory.java

```diff
@@ -68,7 +68,7 @@ private DefaultRemoteManagerFactory(
 
   @Override
   public RemoteManager getInstance(final String newRmName) {
-    return getInstance(newRmName, 0, this.codec, this.errorHandler);
+    return getInstance(newRmName + "1", 0, this.codec, this.errorHandler);
```

Comment:
What is this '+ "1"' for?

Reply:
Removed.

---
In reply to: [199969719](https://github.com/sharathmalladi/reef/pull/6#discussion_r199969719) [](ancestors = 199969719)